Applications/Games/Steam/Online/script.js

```diff
@@ -2,7 +2,6 @@ const OnlineInstallerScript = include("engines.wine.quick_script.online_installe
 const { LATEST_STAGING_VERSION } = include("engines.wine.engine.versions");
 
 const Corefonts = include("engines.wine.verbs.corefonts");
-const WindowsVersion = include("engines.wine.plugins.windows_version");
```

Comment:
I think this include can be removed in the SteamScript as well.

Reply:
No because there is an uplay fix that uses this include.